source/administration/important-upgrade-notes.rst

```diff
@@ -13,6 +13,8 @@ Important Upgrade Notes
 |                                                    | Image link and YouTube previews do not display unless **System Console > Enable Link Previews** is enabled. Please ensure that your Mattermost server is         |
 |                                                    | connected to the internet and has network access to the websites from which previews are expected to appear.                                                     |
 |                                                    | `Learn more here <https://forum.mattermost.org/t/link-previews-managed-server-side-in-v5-12-and-later/7712>`_.                                                   | 
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | ``ExperimentalEnablePostMetadata`` setting was removed.                                                                                                          |
```

Comment:
``ExperimentalEnablePostMetadata`` setting was removed. Post metadata, including post dimensions, is now stored in the database to correct scroll position and eliminate scroll jumps as content loads in a channel.